src/hermezjs-typings.d.ts

```diff
@@ -191,34 +190,34 @@ declare module "@hermeznetwork/*" {
   //   forgedTransactions: number;
   // }
 
-  // type Coordinator = HermezApiResourceItem & {
-  //   bidderAddr: string;
-  //   forgerAddr: string;
-  //   ethereumBlock: number;
-  //   URL: string;
-  // }
+  type Coordinator = HermezApiResourceItem & {
+    // bidderAddr: string;
+    forgerAddr: string;
+    // ethereumBlock: number;
+    URL: string;
+  };
 
-  // interface Period {
-  //   slotNum: number;
-  //   fromBlock: number;
-  //   toBlock: number;
-  //   fromTimestamp: string;
-  //   toTimestamp: string;
-  // }
+  interface Period {
+    // slotNum: number;
+    // fromBlock: number;
+    // toBlock: number;
```

Comment:
Why all these commented properties? Isn't it better to just remove them?

Reply:
I thought of adding all the props present on the entities of the lib and leave commented and not implemented those that are not currently used by the wallet. On each PR, I uncomment some new props that become used by the new modules I refactor. The idea is that when we end the refactor we will have a clear picture of what props do we use and what don't.

At that point, we could opt to remove the commented features or implement them all since, at the end of the day, these types definition module should not belong to the app, but to the lib. This module describes the types of the library and I think that eventually should cover all the features of the library, no matter whether they are used on the wallet on not. 

Since it's a library, I think some other projects using it may require some other stuff. Does it make sense?

Reply:
Makes sense. Probably worth mentioning these decisions in the PR itself in the future